src/hooks/login/useWebWalletLogin.ts

```diff
@@ -53,7 +52,7 @@ export const useWebWalletLogin = ({
         expires: expires
       };
 
-      if (hasNativeAuth && !token) {
```

Comment:
this was correct, it should not fetch a native auth token if hasNativeAuth flag is not set